extra/man/alacritty.5.scd

```diff
@@ -996,7 +996,7 @@ relied upon.
 	Example:
 		_ALACRITTY_EXTRA_LOG_TARGETS="winit;vte" alacritty -vvv_
 
-*renderer* = _"glsl3"_ | _"gles2"_ | _"gles2_pure"_ | _"None"_
+*renderer* = _"glsl3"_ | _"gles2"_ | _"gles2pure"_ | _"None"_
```

Comment:
can't we bring back how it was?

Reply:
We could, but that would require proc macro changes. Not sure that's worth it for a debug option?